lib/install/validate/storage.go

```diff
@@ -53,16 +53,27 @@ func (v *Validator) storage(ctx context.Context, input *data.Data, conf *config.
 		conf.AddImageStore(imageDSpath)
 	}
 
-	if conf.VolumeLocations == nil {
-		conf.VolumeLocations = make(map[string]*url.URL)
-	}
-
-	// TODO: add volume locations
+	//volume location population
 	for label, volDSpath := range input.VolumeLocations {
-		dsURL, _, err := v.DatastoreHelper(ctx, volDSpath, label, "--volume-store")
-		v.NoteIssue(err)
-		if dsURL != nil {
-			conf.VolumeLocations[label] = dsURL
+
+		switch label {
```

Comment:
The 2 cases are almost completely identical.  Can we nuke the `switch`, and use a simple `if {} else {} ` for the ` field that's different?